JoyboyCommunity/src/components/Menu/styles.ts

```diff
@@ -28,7 +28,7 @@ export default ThemedStyleSheet((theme) => ({
     position: 'absolute',
     height: 'auto',
     gap: StyleSheet.hairlineWidth,
-    backgroundColor: theme.colors.background,
+    backgroundColor: `${theme.colors.divider} !important`,
```

Comment:
This is invalid React Native style, RN doesn't have !important

Reply:
hey @ugur-eren updated